tests/dataframe_tests/test_data/parent/schema.json

```diff
@@ -0,0 +1,7454 @@
+{
```

Comment:
1. what is different about this new "parent" dictionary?
2. can we not add the use case we need to one of the existing schemas?
3. do we really need a full schema with all these nodes? or could we simplify it to only a couple nodes, just enough to test this use case?

Reply:
we need to change the data to make the new test. Since, in the old data, we don't have the real many_to_many relationship from center to project. Changing that relationship will effect the current test of IBDGC while we don't have qa-ibdgc to regenerate it. It will be more complicated and time consuming to change data for current ibd

Reply:
ok similar to my other comment, it's easier to understand things when there isn't a ton of unrelated data, even if you don't want to change this one now (removing unnecessary nodes/props), i think it would be better to keep that in mind for future tests - try to write minimal testing setups